developers/addons/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: developersguide
+title: Add-ons
+---
+
+# Developing an add-on
```

Comment:
```suggestion
# Developing an Add-on
```

Reply:
Yes, that's due to an incorrect `.editorconfig` in this repo (it defines CRLF instead of LF which is actually used).

Reply:
This should be fine now.